rpc/client.go

```diff
@@ -236,17 +236,42 @@ func (c *Client) Stop() {
 
 func (c *Client) monitorHealth(ctx context.Context, statusCh chan struct{}) {
 	sendFullStatusEventFunc := func() {
+		if c.walletFeed == nil {
+			return
+		}
+
 		blockchainStatus := c.healthMgr.GetFullStatus()
 		encodedMessage, err := json.Marshal(blockchainStatus)
 		if err != nil {
 			c.log.Warn("could not marshal full blockchain status", "error", err)
 			return
 		}
+
+		c.walletFeed.Send(walletevent.Event{
+			Type:    EventBlockchainHealthChanged,
+			Message: string(encodedMessage),
+			At:      time.Now().Unix(),
+		})
+	}
+
+	sendShortStatusEventFunc := func() {
```

Comment:
This should be removed after fixing https://github.com/status-im/status-mobile/issues/21401
and the corresponding ticket on the desktop.